### PR TITLE
RavenDB-13812 Studio - Traffic Watch - Filtering by type exclude all …

### PR DESCRIPTION
**Pull request by Danielle9897**
…entries

### CODE REVIEW

File: src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
             </div>
             <div class="flex-horizontal flex-wrap margin-right">
                 <div>
-                    <input class="form-control" placeholder="Filter" data-bind="textInput: filter">
+                    <input class="form-control" placeholder="Filter" data-bind="textInput: filter" title="Filter 'Custom Info' & 'URI' columns">
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html]: can we put that info to placeholder instead to title? If it fits

**Review by ml054 — CHANGES_REQUESTED** [review]

File: src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html]: image

Comment (ml054) [reply src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html]: please make input wider?

File: src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html [hunk 2]
```diff
@@ -8,29 +8,27 @@
...
-                <div>
-                    <input class="form-control" placeholder="Filter" data-bind="textInput: filter" title="Filter 'Custom Info' & 'URI' columns">
-                </div>
                 <div class="flex-horizontal">
                     <div class="btn-group">
                         <select id="visibleTypesSelector" size="5" multiple="multiple"
                                 data-bind="options: filteredTypeData, selectedOptions: selectedTypeNames, optionsText: 'propertyName', optionsValue: 'propertyName'"></select>
                     </div>
+                    <div id="trafficFilter">
+                        <input class="form-control" placeholder="Filter 'Custon Info' & 'URI' Columns" data-bind="textInput: filter" title="Filter 'Custom Info' & 'URI' Columns">
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html]: no need for title no - we have placeholder

**Review by ml054 — CHANGES_REQUESTED** [review]

File: src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html]: done - title removed

File: src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html]: done

**Review by ml054 — APPROVED** [review]

**Pull request merged**